server/src/main/java/jass/server/util/GameUtil.java

```diff
@@ -161,6 +181,16 @@ public void onChosenGameMode(final ChosenGameModeData data) {
             }
             RoundRepository.getSingleton(null).update(currentRound);
             broadcast(broadcastGameMode);
+
+            // send turn information to clients
+            TurnEntity turn = addNewTurn(client.getUser(), currentRound);
+            TurnRepository.getSingleton(null).add(turn);
```

Comment:
turn seems to already get added to TurnReposiory in function addNewTurn(). Hence with this line of code the turn is added twice to Repository